fix failing unit tests in `gradle-plugin/openshift` module

### Component

OpenShift Gradle Plugin

### Task description

### Description
Related to https://github.com/eclipse/jkube/issues/1338

⚠️ A windows machine is required to reproduce and fix problems in this issue

When executing `mvn clean install` on Windows, the following unit tests are failing in `gradle-plugin/openshif` module:

```
[ERROR] Failures:
[ERROR]   OpenShiftHelmPushTaskTest.runTask_withNoTemplateDir_shouldThrowException:62
Expecting throwable message:
  "C:\Users\rokum\AppData\Local\Temp\junit5048454430597425837\build\classes\java\main\META-INF\jkube\openshift"
to contain:
  "META-INF/jkube/openshift"
but did not.

[ERROR]   OpenShiftHelmTaskTest.runTask_withNoTemplateDir_shouldThrowException:61
Expecting throwable message:
  "C:\Users\rokum\AppData\Local\Temp\junit5459887138586727047\build\classes\java\main\META-INF\jkube\openshift"
to contain:
  "META-INF/jkube/openshift"
but did not.

Throwable that failed the check:
```

### Expected Behavior
All unit tests in `gradle-plugin/openshift` module should pass on windows

### Acceptance Criteria
- [ ] Build completes successfully on windows for `gradle-plugin/openshift` module

--- a/gradle-plugin/openshift/src/test/java/org/eclipse/jkube/gradle/plugin/task/OpenShiftHelmPushTaskTest.java
+++ b/gradle-plugin/openshift/src/test/java/org/eclipse/jkube/gradle/plugin/task/OpenShiftHelmPushTaskTest.java
@@ -23,6 +23,7 @@ import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.RegisterExtension;
 import org.mockito.MockedConstruction;
 
+import java.io.File;
 import java.io.IOException;
 import java.nio.file.NoSuchFileException;
 
@@ -59,7 +60,7 @@ class OpenShiftHelmPushTaskTest {
     // When & Then
     assertThatIllegalStateException()
         .isThrownBy(openShiftHelmPushTask::runTask)
-        .withMessageContaining("META-INF/jkube/openshift")
+        .withMessageContaining("META-INF" + File.separator + "jkube" + File.separator + "openshift")
         .withCauseInstanceOf(NoSuchFileException.class);
   }
 

--- a/gradle-plugin/openshift/src/test/java/org/eclipse/jkube/gradle/plugin/task/OpenShiftHelmTaskTest.java
+++ b/gradle-plugin/openshift/src/test/java/org/eclipse/jkube/gradle/plugin/task/OpenShiftHelmTaskTest.java
@@ -22,6 +22,7 @@ import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.RegisterExtension;
 import org.mockito.MockedConstruction;
 
+import java.io.File;
 import java.io.IOException;
 import java.nio.file.NoSuchFileException;
 
@@ -58,7 +59,7 @@ class OpenShiftHelmTaskTest {
     // When & Then
     assertThatIllegalStateException()
         .isThrownBy(kubernetesHelmTask::runTask)
-        .withMessageContaining("META-INF/jkube/openshift")
+        .withMessageContaining("META-INF" + File.separator + "jkube" + File.separator + "openshift")
         .withCauseInstanceOf(NoSuchFileException.class);
   }
 

--- a/openshift-maven-plugin/plugin/src/test/java/org/eclipse/jkube/maven/plugin/mojo/build/OpenShiftResourceMojoTest.java
+++ b/openshift-maven-plugin/plugin/src/test/java/org/eclipse/jkube/maven/plugin/mojo/build/OpenShiftResourceMojoTest.java
@@ -85,7 +85,7 @@ class OpenShiftResourceMojoTest {
     final File generatedArtifact = new File(resourceMojo.targetDir, "openshift.yml");
     assertThat(generatedArtifact)
       .exists()
-      .content().isEqualTo("---\napiVersion: v1\nkind: List\n");
+      .content().isEqualTo(String.format("---%napiVersion: v1%nkind: List%n"));
     verify(resourceMojo.projectHelper, times(1))
       .attachArtifact(resourceMojo.project, "yml", "openshift", generatedArtifact);
   }

